Source/Extensions/Blazorise.DataGrid/DataGrid.razor.cs

```diff
@@ -494,7 +494,7 @@ public void FilterData()
         /// </summary>
         /// <param name="fieldName">Cell field name.</param>
         /// <param name="value">New cell value.</param>
-        public void UpdateCell( string fieldName, object value )
+        public void UpdateEditCell( string fieldName, object value )

```

Comment:
To be more precise could be, `UpdateEditCellValue`, as it's always the value of the cell, not other cell properties. It is no big deal, I think it's still quite clear to the developer.

Reply:
Hmm tough choice...

Reply:
yep, haha naming is hard! :D

Reply:
I will leave it as it is. No need to make it too long :)